Kudu.Core/Infrastructure/SecurityUtility.cs

```diff
@@ -39,6 +39,13 @@ public static string DecryptSecretString(string content)
             return protector.Unprotect(content);
         }
 
+        public static string GenerateFunctionToken()
+        {
+            string siteName = ServerConfiguration.GetApplicationName();
+            string issuer = $"https://{siteName}.scm.azurewebsites.net";
```

Comment:
Note that on other deployments, the host name is not azurewebsites.net. It's probably not a big deal, since I don't think this is used as an actual URL. Does it even need to be a url? /cc @fabiocav